kernel/configs/README.md

```diff
@@ -1,21 +1,60 @@
+## Types of config files
+
+This directory holds config files for the Kata Linux Kernel in two forms:
+
+- As complete config files that can be used as-is.
+- A tree of config file 'fragments' in the `fragments` sub-folder, that are
+  constructed into a complete config file using the kernel
+  `scripts/kconfig/merge_config.sh` script.
+
 ## How to use config files
 
-config files must be copied in the kernel source code directory and renamed to `.config`
+Complete config files must be copied in the kernel source code directory and renamed to `.config`
 
 For example:
 
+```bash
+$ cp x86_kata_kvm_4.14.x linux-4.14.22/.config
+$ pushd linux-4.14.22
+$ make ARCH=x86_64 -j4
 ```
-cp x86_kata_kvm_4.14.x linux-4.14.22/.config
-pushd linux-4.14.22
-make ARCH=x86_64 -j4
+
+Alternatively, use the [`build_kernel.sh`](../build-kernel.sh) script to set up the
+config file, for example:
+
+```bash
+$ ./build-kernel.sh setup
 ```
 
+The `build_kernel.sh` is also the recommended way to construct `.config` files from the fragments.
+
+Run `./build-kernel.sh help` for more information.
+
 ## How to modify config files
 
+Complete config files can be modified either with an editor, or more easily
+using the kernel `Kconfig` configuration tools, for example:
+
```

Comment:
Not strictly part of this PR, but would be good to fix the command block below on a separate commit to add the prompt for compliance with https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.

Reply:
done. folded it in.